pkg/operator/k8sutil/k8sutil.go

```diff
@@ -63,3 +65,21 @@ func GetK8SVersion(clientset kubernetes.Interface) (*version.Version, error) {
 	}
 	return version.MustParseSemantic(serverVersion.GitVersion), nil
 }
+
+// Hash MD5 hash a given string
+func Hash(s string) string {
+	return fmt.Sprintf("%x", md5.Sum([]byte(s)))
+}
+
+// TruncateNodeName hashes the nodeName in case it would case the name to be longer than 63 characters
+// WARNING If your format and nodeName as a hash, are longer than 63 chars it won't be truncated!
+// Your format alone should only be 31 chars at max because of MD5 hash being 32 chars.
+// For more information, see the following resources:
+// https://stackoverflow.com/a/50451893
+// https://stackoverflow.com/a/32294443
+func TruncateNodeName(format, nodeName string) string {
+	if len(nodeName)+len(fmt.Sprintf(format, "")) > validation.DNS1035LabelMaxLength {
```

Comment:
To help troubleshoot, how about if we just log what the node name is and what it was translated to? When needed during troubleshooting, we could also find an online MD5 hash generator to translate a node name if the logs aren't available.

Reply:
Sure, I'll add that tomorrow morning.